.travis.yml

```diff
@@ -4,10 +4,7 @@ php:
   - '5.6'
   - '7.0'
 install:
-  - composer install
-before_script:
-  - curl -s http://getcomposer.org/installer | php
-  - php composer.phar install --dev --no-interaction
+  - composer update --no-interaction
```

Comment:
good catch about not needing to install composer in travis. Let's keep this as a composer install instead of an update (this goes along with keeping composer.lock in VC).